### ISSUE TITLE
Problems encountered during execution

### ISSUE THREAD

**Issue by ZWXCV** [StZHY/KGAN#9]
Hello, the following error occurred during the execution of the source code you released:
2024-05-09 14:20:54.361804: I tensorflow/stream executor/platform/default/dso loader. c:4] Sucesfully opened dvamic library libcublas. so. 10.0Traceback(most recent call last):
File "main.py", line 29, in<module>train(args, data info, args.show loss)
File "/root/KGAN-master/src/train.py", line 34,in traintrain auc, train acc = evaluation(sess, args, model, train data, aggregate set, relation set, args. batch size)
File "/root/KGAN-master/src/train.py", line 77, in evaluationauc, acc = model. eval(sess, get feed dict(args, model, data, aggregate set, relation set, start, start + batch size))
TypeError:,numpy.float64 object is not iterable
Is there a problem with the code logic, but I don't see it? Or is there something wrong with the format of the book dataset? I hope the author can give some suggestions and look forward to your reply.
